2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_setup.py:_flush():76] Loading settings from C:\Users\jaydenpark\.config\wandb\settings
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_setup.py:_flush():76] Loading settings from C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\settings
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_init.py:_log_setup():428] Logging user logs to C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\run-20230314_140010-30vjiqkl\logs\debug.log
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_init.py:_log_setup():429] Logging internal logs to C:\Users\jaydenpark\Documents\GitHub\disaster_tweet\wandb\run-20230314_140010-30vjiqkl\logs\debug-internal.log
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_init.py:_jupyter_setup():378] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x0000021B8B351460>
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_init.py:init():462] calling init triggers
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_init.py:init():465] wandb.init called with sweep_config: {}
config: {'learning_rate': 1e-06, 'data_size': 1, 'batch_size': 128}
2023-03-14 14:00:10,034 INFO    MainThread:30444 [wandb_init.py:init():515] starting backend
2023-03-14 14:00:10,034 INFO    MainThread:30444 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2023-03-14 14:00:10,034 INFO    MainThread:30444 [backend.py:ensure_launched():217] starting backend process...
2023-03-14 14:00:10,118 INFO    MainThread:30444 [backend.py:ensure_launched():222] started backend process with pid: 22564
2023-03-14 14:00:10,119 INFO    MainThread:30444 [wandb_init.py:init():525] backend started and connected
2023-03-14 14:00:10,213 INFO    MainThread:30444 [wandb_run.py:_label_probe_notebook():1089] probe notebook
2023-03-14 14:00:10,213 INFO    MainThread:30444 [wandb_run.py:_label_probe_notebook():1099] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-03-14 14:00:10,213 INFO    MainThread:30444 [wandb_init.py:init():596] updated telemetry
2023-03-14 14:00:10,213 INFO    MainThread:30444 [wandb_init.py:init():627] communicating run to backend with 30 second timeout
2023-03-14 14:00:11,610 INFO    MainThread:30444 [wandb_run.py:_on_init():1924] communicating current version
2023-03-14 14:00:11,662 INFO    MainThread:30444 [wandb_run.py:_on_init():1928] got version response upgrade_message: "wandb version 0.13.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2023-03-14 14:00:11,662 INFO    MainThread:30444 [wandb_init.py:init():662] starting run threads in backend
2023-03-14 14:00:11,945 INFO    MainThread:30444 [wandb_run.py:_console_start():1898] atexit reg
2023-03-14 14:00:11,945 INFO    MainThread:30444 [wandb_run.py:_redirect():1771] redirect: SettingsConsole.WRAP
2023-03-14 14:00:11,945 INFO    MainThread:30444 [wandb_run.py:_redirect():1808] Wrapping output streams.
2023-03-14 14:00:11,946 INFO    MainThread:30444 [wandb_run.py:_redirect():1832] Redirects installed.
2023-03-14 14:00:11,947 INFO    MainThread:30444 [wandb_init.py:init():688] run started, returning control to user process
2023-03-14 14:00:11,947 INFO    MainThread:30444 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:00:11,947 INFO    MainThread:30444 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:00:11,952 INFO    MainThread:30444 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:00:11,956 INFO    MainThread:30444 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:00:11,956 INFO    MainThread:30444 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:00:11,959 INFO    MainThread:30444 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:00:12,148 INFO    MainThread:30444 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:00:12,148 INFO    MainThread:30444 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:00:12,163 INFO    MainThread:30444 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:00:19,573 INFO    MainThread:30444 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:00:19,573 INFO    MainThread:30444 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:00:19,591 INFO    MainThread:30444 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:00:21,726 INFO    MainThread:30444 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:00:21,726 INFO    MainThread:30444 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:00:21,743 INFO    MainThread:30444 [wandb_init.py:_resume_backend():350] resuming backend
2023-03-14 14:00:27,528 INFO    MainThread:30444 [wandb_run.py:_config_callback():1136] config_cb None None {'vocab_size': 30522, 'learning_rate': 0.0003}
2023-03-14 14:00:31,925 INFO    MainThread:30444 [wandb_init.py:_pause_backend():341] pausing backend
2023-03-14 14:00:31,925 INFO    MainThread:30444 [jupyter.py:save_ipynb():377] not saving jupyter notebook
2023-03-14 14:00:42,826 INFO    MainThread:30444 [wandb_run.py:_atexit_cleanup():1867] got exitcode: 0
2023-03-14 14:00:42,827 INFO    MainThread:30444 [wandb_run.py:_restore():1839] restore
